src/main/java/me/pseudoknight/chdiscord/MemberFunctions.java

```diff
@@ -54,7 +54,7 @@ public String getName() {
 		}
 
 		public String docs() {
-			return"void {member, string} Get the server nickname for a member."
+			return"void {member} Get the server nickname for a member."
```

Comment:
This should also document the return type as "string". (eg. "string {member} Get ...")

Reply:
Good catch thanks, I'll fix that.